tests/e2e/e2e_test.go

```diff
@@ -51,11 +50,10 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 	subnets := vms.XSVMSubnetsOrPanic(nodes...)
 
 	upgrades := upgrade.Default
-	if flagVars.ActivateGranite() {
-		upgrades.GraniteTime = upgrade.InitiallyActiveTime
-		upgrades.GraniteEpochDuration = 4 * time.Second
```

Comment:
By removing this, we are changing the epoch duration from `4s` to `30s` in our tests. This causes some of our E2E tests to sleep longer...

I think this is fine... The CI time seems to be around 15m-16m before and after this change... So doesn't seem like a significant slowdown... But if we wanted to keep it the same, we should override the epoch duration regardless of whether Helicon is activated.

Reply:
I will keep the name